spec/pipeline_spec.rb

```diff
@@ -96,21 +96,13 @@ def call
   context 'when there is an error' do
     subject do
       NxtPipeline::Pipeline.new do |pipeline|
-        pipeline.constructor(:service) do |step, arg|
+        pipeline.constructor(:service, to_s: -> (step) { step.service_class.to_s }) do |step, arg|
```

Comment:
What about making the step class name the default `to_s` value (as it is right now)?

Reply:
Forget about it, having more info probably is good.

Reply:
Also there is not such thing as step class name. Constructors can vary, the step class will always be the same.